packages/manager/src/features/ObjectStorage/AccessKeyLanding/AccessKeyTable/HostNameTableCell.tsx

```diff
@@ -27,27 +27,37 @@ export const HostNameTableCell = (props: Props) => {
   if (!regionsLookup || !regionsData || !regions || regions.length === 0) {
     return <TableCell>None</TableCell>;
   }
-  const label = regionsLookup[storageKeyData.regions[0].id]?.label;
-  const s3Endpoint = storageKeyData?.regions[0]?.s3_endpoint;
-  const endpointType = storageKeyData?.regions[0]?.endpoint_type;
+  const formatEndpoint = (region: ObjectStorageKeyRegions) => {
+    const label = regionsLookup[region.id]?.label;
+    const endpointType = region.endpoint_type
+      ? ` (${region.endpoint_type})`
+      : '';
+    return `${label}${endpointType}: ${region.s3_endpoint}`;
+  };
+
+  const firstRegion = regions[0];
+  const formattedFirstEndpoint = formatEndpoint(firstRegion);
+  const allEndpoints = regions.map(formatEndpoint).join('\n');
+  const showMultipleRegions = regions.length > 1;
 
   return (
     <TableCell>
-      {label}
-      {endpointType && ` (${endpointType})`}: {s3Endpoint}&nbsp;
-      {storageKeyData?.regions?.length === 1 && (
-        <StyledCopyIcon text={s3Endpoint} />
-      )}
-      {storageKeyData.regions.length > 1 && (
-        <StyledLinkButton
-          onClick={() => {
-            setHostNames(storageKeyData.regions);
-            setShowHostNamesDrawers(true);
-          }}
-          type="button"
-        >
-          and {storageKeyData.regions.length - 1} more...
-        </StyledLinkButton>
+      {formattedFirstEndpoint}&nbsp;
+      {showMultipleRegions ? (
+        <>
+          | +{regions.length - 1} regions |&nbsp;
```

Comment:
Optional: maybe we could use `pluralize` so it can say `+1 region` instead of `+1 regions` 

```suggestion
          | +{regions.length - 1} {pluralize('region', 'regions', regions.length - 1)} |&nbsp;
```

Reply:
Not blocking, but the wrapping on small screen looks a bit messy

![Screenshot 2024-12-04 at 3 27 17 PM](https://github.com/user-attachments/assets/2caed4ec-0453-4f5c-8d98-bc9523a04a95)